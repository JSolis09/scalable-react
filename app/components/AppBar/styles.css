.appBar {
    width: 100%;
    padding: 20px;
    background-color: #00afcc;
    color: #fff;
    font-size: 24px;
    display: flex;
    justify-content: space-between;
}

.appBar .iconButton {
    cursor: pointer;
    margin-right: 15px;
}

.appBar .icon {
    font-size: 20px;
    color: #fff;
}

.appBar .icon:hover {
    color: #e5e5e5;
}

.linkContainer {
    font-size: 16px;
    margin-top: 6px;
}

.appBar .heading {
    text-align: left;
    display: flex;
    flex-grow: 2;
    color: #fff;
}