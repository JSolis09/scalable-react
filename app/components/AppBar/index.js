import React from 'react';
import FontAwesome from 'react-fontawesome';
import './styles.css';

function AppBar() {
  return (
    <div className="appBar">
      <div className="iconButton">
        <FontAwesome className="icon" name="bars" />
      </div>
      <div className="heading">
        Coder daily
      </div>
      <div className="linkContainer">
        Log in
      </div>
    </div>
  );
}

AppBar.propTypes = {};

export default AppBar;
