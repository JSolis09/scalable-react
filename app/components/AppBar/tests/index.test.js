import React from 'react';
import { shallow } from 'enzyme';

import AppBar from '../index';

describe('<AppBar />', () => {
  it('should render correctly', () => {
    const appBarComponent = shallow(<AppBar />);
    expect(appBarComponent).toMatchSnapshot();
  });
});
