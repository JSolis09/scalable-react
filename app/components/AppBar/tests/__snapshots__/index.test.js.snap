// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render correctly 1`] = `
<div
  className="appBar"
>
  <div
    className="iconButton"
  >
    <FontAwesome
      className="icon"
      name="bars"
    />
  </div>
  <div
    className="heading"
  >
    Coder daily
  </div>
  <div
    className="linkContainer"
  >
    Log in
  </div>
</div>
`;
